crates/rust-analyzer/src/cli.rs

```diff
@@ -22,16 +22,10 @@ pub enum Verbosity {
 
 impl Verbosity {
     pub fn is_verbose(self) -> bool {
-        match self {
-            Verbosity::Verbose | Verbosity::Spammy => true,
-            _ => false,
-        }
+        matches!(self, Verbosity::Verbose | Verbosity::Spammy)
     }
     pub fn is_spammy(self) -> bool {
-        match self {
-            Verbosity::Spammy => true,
-            _ => false,
-        }
+        matches!(self, Verbosity::Spammy)
```

Comment:
This one is short, but we also don't use it as a part of a bigger expression (like `if`), so an explicit match reads better to me.